src/bin/bench.rs

```diff
@@ -160,6 +187,8 @@ fn main() {
     println!("MSQ mpmc: {}", bench_queue_mpmc(MsQueue::new()));
     println!("Seg mpmc: {}", bench_queue_mpmc(SegQueue::new()));
 
+    println!("SpscBufferQueue: {}", bench_spsc());
```

Comment:
It'd be great to make this a generic bench like the others, so that the numbers can be compared across the data structures.